tests/php/WordPressThemeTest.php

```diff
@@ -6,12 +6,27 @@
 
 final class WordPressThemeTest extends DOMTestCase
 {
+    private static $AMO_BASE_URL;
     private static $BUILD_DIR;
 
     public static function setUpBeforeClass(): void
     {
+        self::$AMO_BASE_URL = $_ENV['AMO_BASE_URL']
+            ? $_ENV['AMO_BASE_URL']
+            : 'https://addons.mozilla.org';
         self::$BUILD_DIR = realpath(__DIR__ . '/../../build/');
 
+        fwrite(
+            STDERR,
+            print_r(
+                sprintf(
+                    'Running with AMO_BASE_URL env variable: %s',
+                    self::$AMO_BASE_URL
+                ),
+                true
+            )
+        );
```

Comment:
Why is this needed?

Reply:
I added this so we can verify that the test did in fact run with the expected environment variable. Without it, there is no way to know if our intent to run the test with an `AMO_BASE_URL` for dev actually happened. I guess we don't need it on CI, after I have verified it for myself locally, but it seemed worth keeping in place.

Reply:
Actually, I can't verify the behaviour on CI locally, so this is still needed if we want to be sure it's working (and there wasn't an error in the CI configuration).

Reply:
mmm how about we use the phpunit config to inject the env var then? We can add the following XML in the [`phpunit.xml.dist`](https://github.com/mozilla/addons-blog/blob/main/phpunit.xml.dist) file:

```xml
<phpunit colors="true" bootstrap="tests/php/bootstrap.php">
  <testsuites>
    <testsuite name="AMO Blog Test Suite">
      <directory>tests/php/</directory>
    </testsuite>
  </testsuites>
  <php>
    <env name="AMO_BASE_URL" value="https://addons-dev.allizom.org" />
  </php>
</phpunit>
```

Reply:
We can also add an assertion on the length of `$_ENV['AMO_BASE_URL']` to prevent empty values.

Reply:
Making that change causes the tests to fail locally, because the env var isn't set for `build:wptheme`. I could add it to the command for `build:wptheme`, but I'm not sure that's what we want. We could also include instructions for running `build:wptheme` with the env var set when testing locally, but that seems a bit complicated to me. We could add another `yarn` script that does what `build:wptheme` does, and includes the env var, and update the instructions to run *that* locally, but again, this seems overly complicated.

I think the set up was better before making that change, as it worked on both CI and locally, without having to force the env var locally.

WDYT?

Reply:
Gaah, I forgot about that, _again_.

Because I don't quite like this code (which isn't too PHP-y), I'd suggest to print the variable in the shell script here: https://github.com/mozilla/addons-blog/blob/9e6e4c37c8f949c7bf0879785df9801b6ddbac89/.circleci/config.yml#L63-L66

Something like:

```sh
echo "AMO_BASE_URL=$AMO_BASE_URL"
```